app/favorites/page.tsx

```
@@ -167,18 +167,19 @@ const FavoritesSection = ({
-  const { searchQuery, searchResults, setCurrentCategory } =
+  const { searchQuery, searchResults, selectedTags, setCurrentCategory } =
     useSearch();
 
   useEffect(() => {
     setCurrentCategory(null);
   }, [setCurrentCategory]);
 
   const displayedFavorites = useMemo(() => {
-    return searchQuery && searchQuery.trim()
+    // Show searchResults when there's a search query OR when tags are selected
+    return (searchQuery && searchQuery.trim()) || selectedTags.length > 0
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)